### PR TITLE
update bslib examples

### LINKED ISSUE #1604: Update bslib variables example in the customization vignette
* font-scale
* navbar colors
* font_face
* 
via https://github.com/rstudio/bslib/pull/288/

### PR DESCRIPTION
**Pull request by maelle**
Fix #1604

### CODE REVIEW

File: vignettes/customization.Rmd [hunk 1]
```diff
@@ -92,6 +92,8 @@ Rather than picking an entire theme, you can customise individual theme componen
 template:
   bootstrap: 4
   bslib:
+    font_scale: 1.2
+    font_link: !expr bslib::font_google("muli")
```
Comment (hadley) [inline vignettes/customization.Rmd:96]: Should use new new syntax that doesn't need `!expr`: https://github.com/rstudio/bslib/issues/256

Comment (cpsievert) [reply vignettes/customization.Rmd:96]: Also it should be `font_base` not `font_link` (I think?)

Comment (maelle) [reply vignettes/customization.Rmd]: Thanks both, I updated the syntax following https://rstudio.github.io/bslib/articles/bslib.html#main-colors-fonts

File: DESCRIPTION [hunk 2]
```diff
@@ -73,4 +73,6 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 SystemRequirements: pandoc
-Remotes: r-lib/downlit
+Remotes:
```
Comment (hadley) [inline DESCRIPTION:76]: Given that this just an example, and bslib will probably be released before pkgdown, I wouldn't bother bumping the required version here.

**Review by hadley — APPROVED** [review]

**Pull request merged**